classes/c++.oeclass

```diff
@@ -38,13 +38,13 @@ SDK_CXXFLAGS		?= "${SDK_CFLAGS} -fpermissive"
 CXXFLAGS			 = "${HOST_CXXFLAGS}"
 HOST_CXXFLAGS:native		 = "${BUILD_CXXFLAGS}"
 HOST_CXXFLAGS:cross		 = "${BUILD_CXXFLAGS}"
-HOST_CXXFLAGS:machine		 = "${MACHINE_CXXFLAGS}"
+HOST_CXXFLAGS:machine		 = "${MACHINE_CXXFLAGS} ${PIC_OPTION}"
 HOST_CXXFLAGS:sdk		 = "${SDK_CXXFLAGS}"
 HOST_CXXFLAGS:sdk-cross		 = "${BUILD_CXXFLAGS}"
 HOST_CXXFLAGS:canadian-cross	 = "${SDK_CXXFLAGS}"
 TARGET_CXXFLAGS:native		 = "${BUILD_CXXFLAGS}"
 TARGET_CXXFLAGS:cross		 = "${MACHINE_CXXFLAGS}"
-TARGET_CXXFLAGS:machine		 = "${MACHINE_CXXFLAGS}"
+TARGET_CXXFLAGS:machine		 = "${MACHINE_CXXFLAGS} ${PIC_OPTION}"
```

Comment:
Why are you adding it to CXXFLAGS and not CFLAGS?  Isn't this change just as relevant to C as to C++?